test/final.test.js

```diff
@@ -8,6 +8,28 @@ const { sleep, getPathToNull } = require('./helper')
 // will be emitted. Let's raise this so we do not scare everybody.
 process.setMaxListeners(100)
 
+test('should show warning for pino.final on node 14+', async ({ equal, end }) => {
+  const major = Number(process.versions.node.split('.')[0])
+  if (major < 14) end()
+
+  const dest = pino.destination({ dest: getPathToNull(), sync: false })
```

Comment:
```suggestion
test('should show warning for pino.final on node 14+', ({ equal, end }) => {
  const major = Number(process.versions.node.split('.')[0])
  if (major < 14) end()

  t.plan(1)
  const dest = pino.destination({ dest: getPathToNull(), sync: false })
```

Reply:
@sameer-coder you missed removing the `async` keyword. I believe this is causing the failure https://github.com/pinojs/pino/runs/4080672051?check_suite_focus=true#step:5:1729

Reply:
@jsumners I think it's because of `if (major < 14) end()`. The test still continues `plan(1)`. I will add a `return` statement there and also remove the `async` keyword.